legion/sdk/legion/sdk/templates/callback-response.html

```diff
@@ -0,0 +1,12 @@
+<html>
+<head>
+    <script>
+        document.addEventListener("DOMContentLoaded", function (event) {
+            window.close();
```

Comment:
When I login with an empty config, my browser window doesn't close automatically.

Reply:
What is your browser?

Reply:
Fixed